src/Forms/Form.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme } from '../Theme';
+
+export interface FormProps {
+  /** Text or form components to be displayed */
+  children: ReactNode;
+  /** The id of the form */
+  id?: string;
+  /** The test id of the form */
+  dataTestId?: string;
```

Comment:
Per note on test, we should avoid using `test-id` if there's a valid `role` we can use instead

Reply:
I ended up querying for the label instead as a result of the note above.